traitsui/testing/tester/wx/helpers.py

```diff
@@ -11,16 +11,78 @@
 
 import wx
 
+from traitsui.testing.tester.exceptions import Disabled
+from traitsui.wx.key_event_to_name import key_map as _KEY_MAP
+
+
+def apply_modifiers(key_event, modifiers):
+    possible_modifiers = ["Alt", "Ctrl", "Meta", "Shift"]
+    modifier_map = {mod: (mod in modifiers) for mod in possible_modifiers}
+    key_event.SetAltDown(modifier_map["Alt"])
+    key_event.SetControlDown(modifier_map["Ctrl"])
+    key_event.SetMetaDown(modifier_map["Meta"])
+    key_event.SetShiftDown(modifier_map["Shift"])
+
+
+def key_click(widget, key, delay=0):
+    """ Performs a key click of the given key on the given widget after
+    a delay.
+
+    Parameters
+    ----------
+    widget : wxObject
```

Comment:
This function requires a `wx.TextCtrl` to use `EmulateKeyPress`. Let's be specific about this type requirement.